lib/python/picongpu/plugins/energy_histogram.py

```diff
@@ -111,46 +115,74 @@ def get(self, species, species_filter="all", iteration=None,
             (defined in ``particleFilters.param``)
         iteration : (unsigned) int [unitless]
             The iteration at which to read the data.
-            @TODO also allow lists here
+            A list of iterations is allowed as well.
+            ``None`` refers to the list of all available iterations.
         include_overflow : boolean, default: False
             Include overflow and underflow bins as the first/last bins.
 
         Returns
         -------
-        energies : np.array of dtype float [unitless]
+        counts : np.array of dtype float [unitless]
             count of particles in each bin
-            @todo if iteration is a list, return a dict
+            If iteration is a list, returns (ordered) dict with iterations as
```

Comment:
QUESTION:
why did you write the adjective `ordered` in parenthesis? 
It is always a dict of type `collections.OrderedDict`.

Reply:
yes, I just felt like saying it's a dict and "btw ordered". Shall I be more explicit? I had the feeling the `OrderedDict` type might confuse people but... dunno.

Reply:
that's fine with me